Chapter3_exercise#2/Chapter3_exercise#2/ViewController.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  ViewController.swift
+//  Chapter3_exercise#2
+//
+//  Created by Andreea Sburlea on 4/7/22.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Do any additional setup after loading the view.
+    }
+    @IBAction func showMessage2(sender: UIButton){
+        let emojiDict = ["👾" : "Alien", "👻" : "Ghost", "🤓" : "Nerd", "🤖" : "Robot"]
+        let wordToLookup = "👾"
+        let meaning = emojiDict[wordToLookup]
+     
+        
+        let selectedButton = sender
```

Comment:
You can directly use the **sender** variable, there is not need to create a new one